### DISCUSSION TITLE
Ideas lol (wk2)

### DISCUSSION THREAD

**Discussion by kk5dire** [kk5dire/radiofnfthing#3] [Suggestions]
put ideas here lol
Good things to include 

- Characters lol
- music ofc
- bpm dif

**Comment by Sammie6789** [comment]
no more poopoo engine tysm  🙈 🙈 🙈

**Comment by spacemaine** [comment]
make week 2 a cake lmao

**Reply by kk5dire** [reply]
can you include 
- Characters 
- music 
- bpm dif 
etc helps lol

**Reply by spacemaine** [reply]
make week 2 cake with portal 2 music

**Reply by kk5dire** [reply]
@spacemain-is-awesome can you give specific tracks please and this would be in this mod correct lol

**Reply by spacemaine** [reply]
https://www.youtube.com/watch?v=dT07bexTkak&list=PLC32EFD96F3E72D4E&index=30

**Reply by Sammie6789** [reply]
man hates me, cannot stand genuine music

**Reply by spacemaine** [reply]
actually this better https://www.youtube.com/watch?v=u9AyaFphAAU

**Reply by kk5dire** [reply]
> https://www.youtube.com/watch?v=dT07bexTkak&list=PLC32EFD96F3E72D4E&index=30

can you give me more specific tracks one for each song in game and a specific asset that you want used for the cake, *and if you want any menu/bar assets you want done etc **(optional)***

**Reply by spacemaine** [reply]
i want those 2 on week 2, also this is a shitpost mod so why you being so specific

**Reply by Sammie6789** [reply]
also why poopoo inputs??

**Reply by spacemaine** [reply]
> > https://www.youtube.com/watch?v=dT07bexTkak&list=PLC32EFD96F3E72D4E&index=30
> 
> can you give me more specific tracks one for each song in game and a specific asset that you want used for the cake, _and if you want any menu/bar assets you want done etc **(optional)**_

it's a portal 2 thing

**Reply by kk5dire** [reply]
i just mean as in their are multiple songs (easier to have multiple known songs for them)  and asset wise same thing maskes it easier and to make sure its as wanted lol , if you really want i could go on google look up portal cake and portal music and call it a day :shipit:

**Reply by Sammie6789** [reply]
a true masterpiece deserve master quality material, yes

**Reply by spacemaine** [reply]
> i just mean as in their are multiple songs (easier to have multiple known songs for them) and asset wise same thing maskes it easier and to make sure its as wanted lol , if you really want i could go on google look up portal cake and portal music and call it a day :shipit:

https://www.pngjoy.com/preview/e7f4k2u5h3y1a3_portal-cake-team-fortress-2-cake-hd-png/

**Reply by kk5dire** [reply]
> > i just mean as in their are multiple songs (easier to have multiple known songs for them) and asset wise same thing maskes it easier and to make sure its as wanted lol , if you really want i could go on google look up portal cake and portal music and call it a day :shipit:
> 
> https://www.pngjoy.com/preview/e7f4k2u5h3y1a3_portal-cake-team-fortress-2-cake-hd-png/

i can work with that yes if thats all for assets, do you want me myself to choose a third song or do you have one in mind l :octocat:

**Reply by spacemaine** [reply]
> > > i just mean as in their are multiple songs (easier to have multiple known songs for them) and asset wise same thing maskes it easier and to make sure its as wanted lol , if you really want i could go on google look up portal cake and portal music and call it a day :shipit:
> > 
> > 
> > https://www.pngjoy.com/preview/e7f4k2u5h3y1a3_portal-cake-team-fortress-2-cake-hd-png/
> 
> i can work with that yes if thats all for assets, do you want me myself to choose a third song or do you have one in mind l :octocat:

week 2's 3rd song is unused so why lol

**Reply by Sammie6789** [reply]
whyy

**Reply by Sammie6789** [reply]
mean

**Reply by Sammie6789** [reply]
ihateyouyoustupidjerk

**Reply by spacemaine** [reply]
> ihateyouyoustupidjerk

calm your tits sammi

**Reply by Sammie6789** [reply]
omg this mad is a madman he dont love me at all whyyyy

**Reply by Sammie6789** [reply]
daddy stoop

**Reply by kk5dire** [reply]
> > > > i just mean as in their are multiple songs (easier to have multiple known songs for them) and asset wise same thing maskes it easier and to make sure its as wanted lol , if you really want i could go on google look up portal cake and portal music and call it a day :shipit:
> > > 
> > > 
> > > https://www.pngjoy.com/preview/e7f4k2u5h3y1a3_portal-cake-team-fortress-2-cake-hd-png/
> > 
> > 
> > i can work with that yes if thats all for assets, do you want me myself to choose a third song or do you have one in mind l :octocat:
> 
> week 2's 3rd song is unused so why lol

easter leg 💯

**Reply by kk5dire** [reply]
@spacemain-is-awesome should i have a super hard song in this one link before lol

**Comment by Sammie6789** [comment]
barbie girl week

**Comment by spacemaine** [comment]
add space-is--awesome to the intro text

**Reply by kk5dire** [reply]
lol ok

**Comment by kk5dire** [comment]
Heya @spacemain-is-awesome would you mind making your week 2 suggestion into an issue, it will help with clarity etc, thanks!

**Reply by spacemaine** [reply]
aight

**Reply by kk5dire** [reply]
thanks!
